Workout/View/fr.lproj/Main.strings

```diff
@@ -0,0 +1,147 @@
+
+/* Class = "UILabel"; text = "[Unit]"; ObjectID = "1Ax-9T-4ZO"; */
+"1Ax-9T-4ZO.text" = "[Unité]";
+
+/* Class = "UILabel"; text = "[y %]"; ObjectID = "8Ip-Rq-vW4"; */
+"8Ip-Rq-vW4.text" = "[y %]";
+
+/* Class = "UILabel"; text = "iPhone"; ObjectID = "ACU-RP-Us6"; */
+"ACU-RP-Us6.text" = "iPhone";
+
+/* Class = "UITableViewSection"; footerTitle = "Select the source from which to load step data. If your devices have not a standard name insert a custom source."; ObjectID = "DE6-lx-y5P"; */
+"DE6-lx-y5P.footerTitle" = "Sélectionnez la source à partir de laquelle charger les données de pas. Si vos appareils n'ont pas de nom, insérez une source personnalisée.";
+
+/* Class = "UILabel"; text = "Label"; ObjectID = "DRM-gr-uTS"; */
+"DRM-gr-uTS.text" = "Label";
+
+/* Class = "UILabel"; text = "Authorize Health data access"; ObjectID = "DeD-yh-hfG"; */
+"DeD-yh-hfG.text" = "Autoriser l'accès aux données de santé";
+
+/* Class = "UILabel"; text = "bpm"; ObjectID = "DyT-UL-y1f"; */
+"DyT-UL-y1f.text" = "bpm";
+
+/* Class = "UILabel"; text = "[Max 60bpm]"; ObjectID = "ES6-46-UVk"; */
+"ES6-46-UVk.text" = "[60 bpm Max]";
+
+/* Class = "UILabel"; text = "%"; ObjectID = "HYe-hx-Xiu"; */
+"HYe-hx-Xiu.text" = "%";
+
+/* Class = "UILabel"; text = "Source Code"; ObjectID = "Hhm-6m-J7Z"; */
+"Hhm-6m-J7Z.text" = "Code Source";
```

Comment:
Since most title/headings have only the first letter capitalized this should also be like that. At least I would capitalize only the first letter in Italian.
```suggestion
"Hhm-6m-J7Z.text" = "Code source";
```